tests/tests/test-subscription.ts

```diff
@@ -126,49 +133,44 @@ describeWithMoonbeam(
       return tx;
     }
 
+    // Little helper to hack web3 that are not complete.
+    function web3Subscribe(type: "newBlockHeaders"): Subscription<BlockHeader>;
+    function web3Subscribe(type: "pendingTransactions"): Subscription<string>;
+    function web3Subscribe(type: "logs", params: {}): Subscription<Log>;
+    function web3Subscribe(type: "newBlockHeaders" | "pendingTransactions" | "logs", params?: any) {
+      return (context.web3.eth as any).subscribe(...arguments);
+    }
```

Comment:
What is the purpose of this code above? Is it to make sure the result of this function is typed for every type of event?

Reply:
You are right, I mostly meant the Typings is broken.
I defined this web3Subscribe to be able to return the modified Subscription class which allow the use of "once" (which is not declared in the default web3 subscription)